src/v2i-hub/CARMACloudPlugin/manifest.json

```diff
@@ -66,7 +66,12 @@
 		{
 			"key":"WebServicePort",
 			"default":"22222",
-			"description":"Server Port for V2X hub"
+			"description":"Server Port for V2X hub to receive TCM messages"
+		},
+		{
+			"key":"listTCM",
+			"default":"true",
```

Comment:
I guess valid entries are `true` and `false`. Is any other string entry invalid. Maybe want to validate this input or at least mention what valid entries are since this is just a string configuration and other bool configurations like DmsPlugin use "True" instead of "true".

Reply:
addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/430/commits/de663dc61f9c951c9b03d4a4a82e7111a7cad0e5